WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -215,6 +215,15 @@ bitcoin-cli stop
 bitcoind
 ```
 
+### Upgrade Knots
+
+```sh
+bitcoin-cli stop
+sudo apt-get update && sudo apt install --only-upgrade bitcoind
```

Comment:
FTR: `apt install --only-upgrade <package-name>` is the command to upgrade packages. It can be used to upgrade Bitcoin Knots, Tor, Ngnix, ufw, etc. 

In this case if we follow the **UPDATE** procedure described in the top of this document we are told to do `apt upgrade` what upgrades absolutely **all** the packages (the most dangerous thing in the Milky Vay).

Reply:
I never observed any problem with upgrading all packages *on Debian*. Upgrades are mostly conservative and don't break things. Not upgrading sounds more dangerous due to security vulnerabilities being unpatched.